services/textProcessor.ts

```
@@ -103,11 +103,13 @@ export class TextProcessor {
     }
   }
 
-  processSubtree(root: HTMLElement, replacements: ReplacementsMap): void {
+  processSubtree(root: HTMLElement, replacements: ReplacementsMap, depth = Infinity): void {
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)